content/millennium/r4/devices/device.md

```diff
@@ -62,12 +62,9 @@ Search for Devices that meet supplied query parameters:
 
 #### Request
 
-    GET https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Device?patient=4478007
+    curl -i -H "Accept: application/fhir+json" "https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Device?patient=4478007"
```

Comment:
i think we should switch example URLs to fhir-open in this file

Reply:
Updated here: https://github.com/cerner/fhir.cerner.com/pull/429/commits/b1170e2cd28d48d44efa74d2f272231892c8ecf8#diff-04bbe83c35d369961fb0c7e0ec9da3b1R65-R91